### PR TITLE
updated with template and policy examples

### PR DESCRIPTION
**Pull request by bigevilbeard**
@aidevnet @annegentle please can you double check this branch and merge - Thanks!

### CODE REVIEW

File: Cisco-SD-WAN.postman_collection.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"info": {
-		"_postman_id": "f3596695-561c-4159-a998-5eca1f85e0cb",
+		"_postman_id": "eb9ea9da-998c-473c-a45d-eb921b13aabc",
```
Comment (annegentle) [inline Cisco-SD-WAN.postman_collection.json:3]: Can anyone do anything with this `_postman_id` value? As in, should you obfuscate it?

**Review by annegentle — COMMENTED** [review]
One question about the ID in the JSON.

File: Cisco-SD-WAN.postman_collection.json (hunk 1 above)
Comment (bigevilbeard) [reply Cisco-SD-WAN.postman_collection.json:3]: Hmm not sure, using Postman  6.6.0 , when I export a collection to JSON, all the _postman_id fields are regenerated.

**Comment by annegentle** [comment]
Ok,  then looks good to me.

**Pull request merged**